### ISSUE TITLE
bug: animaniacs too slow

### ISSUE THREAD

**Issue by ramtrooper** [snes9xgit/snes9x#408]
Works in 1.56 release but latest git it runs in slow-motion during normal gameplay.

**Comment by jjfmz42015** [comment]
At work right now so cant play alot but decided to test this on latest build from appveyor and its going 60 fps, music sounds on point movement is fluid.
rom tested Animaniacs USA CRC32 CF0F14D2. Am I checking the correct game?

Windows version 64 bit and 32 bit tested.

**Comment by ramtrooper** [comment]
That's correct rom. It's noticeable right when game starts in movie studio lobby.

I used all default settings and got slowdown. If I turn off transparency effects, it runs normal speed. When it's on, very sluggish. But I noticed this now:

https://ci.appveyor.com/project/snes9x/snes9x/build/1.56.2-368 = runs very slow

https://ci.appveyor.com/project/snes9x/snes9x/build/1.56.2-367 = runs okay


So probably some appveyor optimization bug. I think this can be safely closed then if devs think it's a compiler fluke.

**Comment by jjfmz42015** [comment]
Strange just running it again, transparency effects on. vsync on, reduce input lag on and no issues 1.56.2-368

Or I should say Im not noticing. I hope at least one other person tests. If not I will try this at home as well after work and see if results vary.

**Comment by bearoso** [comment]
Works fine here, GTK+.

**Comment by jjfmz42015** [comment]
@bearoso  thanks! Windows 32 and 64 here.

@ramtrooper try a fresh download in a fresh folder?

**Comment by OV2** [comment]
I tried those two appveyor builds, both run the same here.

**Comment by retrotalker** [comment]
I think it's non-issue also. No problems here. Try shutting down your OS?


I've re-created the same latest master build here, so try this also and see if your problem comes back:
https://ci.appveyor.com/project/retrotalker/snes9x/build/1.56.2-25

**Comment by jjfmz42015** [comment]
tested more and no issues noticed still.

my tests have been windows 32 bit and 64 bit using appveyor build 3c761968

Ive used WIndows 7 Home Premium 32 bit, and 64 bit, also used WIndows 7 pro 64 bit and WIndows 10 PRO

All with DX9 installed and xbox 360 wireless controller.

I think this issue should be closed unless @ramtrooper  is still having issues?

**Comment by ramtrooper** [comment]
retrotalker build has no problems. Very smooth. Earlier build 1.56.2-368 still slow but 1.56.2-367 okay.

I think it's no problem then.

**Comment by ramtrooper** [comment]
Still no problems with later official commits. Must be a 1-time thing.
